packages/synapse-react-client/src/components/CardDeck/CardDeck.Desktop.tsx

```diff
@@ -89,13 +89,21 @@ export default function CardDeckDesktop({
               backgroundColor: 'white',
               boxShadow: 'none',
               color: 'primary.main',
+              '&:hover': {
+                backgroundColor: 'primary.main',
+                color: 'white',
+              },
             }),
             ...(cardDeckType === 'cckp' && {
               width: '170px',
               fontSize: '14px',
               boxShadow: 'none',
               borderColor: 'primary.main',
               color: 'primary.main',
+              '&:hover': {
+                backgroundColor: 'primary.main',
+                color: 'white',
+              },
```

Comment:
same idea here, if possible apply the common styles unconditionally to reduce duplicate code